practice-three/src/style/components/button.css

```diff
@@ -1,7 +1,17 @@
-.btn {
+.btn-m {
```

Comment:
First, for normal button, class name should be simple. Ex: .btn

Second, for large size button, should reuse existing style from .btn and custom some styles. Btw, pls clearly a button size by large, lg instead of `l`